src/widgets/health-monitoring/HealthMonitoringCpu.tsx

```diff
@@ -0,0 +1,78 @@
+import { Box, Group, Paper, RingProgress, Text } from '@mantine/core';
+import { IconCpu } from '@tabler/icons-react';
+import { useTranslation } from 'react-i18next';
+
+const HealthMonitoringCpu = ({ info, cpuTemp, fahrenheit }: any) => {
+  const { t } = useTranslation('modules/health-monitoring');
+  const toFahrenheit = (value: number) => {
+    return Math.round(value * 1.8 + 32);
+  };
+
+  interface LoadDataItem {
+    label: string;
+    stats: number;
+    progress: number;
+    color: string;
+  }
+
+  const loadData = [
+    {
+      label: `${t('load')} (1min)`,
+      stats: info.loadAverage['1min'],
+      progress: info.loadAverage['1min'],
+      color: 'teal',
+    },
+    {
+      label: `${t('load')} (5min)`,
+      stats: info.loadAverage['5min'],
+      progress: info.loadAverage['5min'],
+      color: 'blue',
+    },
+    {
+      label: `${t('load')} (15min)`,
+      stats: info.loadAverage['15min'],
+      progress: info.loadAverage['15min'],
+      color: 'red',
+    },
+  ] as const;
+
+  return (
+    <Box m="0.4rem" p="0.2rem">
+      <Group>
+        <IconCpu size={40} />
+        <Text fz="lg" tt="uppercase" fw={700} c="dimmed">
+          {t('cpu.utilization')}: {info.cpuUtilization.toFixed(2)}%
+        </Text>
+        <Text fz="lg" tt="uppercase" fw={700} c="dimmed">
+          {fahrenheit
```

Comment:
There's an already existing function in the Weather widget you could use, "getPerferedUnit".
It works with your current parameters.
Not important though, just in case.
If you do end up using it, you would need to extract it to "tools" and modify the weather widget too so don't worry too much about it.